spec.bs

```diff
@@ -492,8 +513,26 @@ The <dfn element-attr for=fencedframe>allow</dfn> attribute, when specified, det
 in the <{fencedframe}>'s [=fenced navigable container/fenced navigable=] is initialized. Its value
 must be a [=serialized permissions policy=]. [[!PERMISSIONS-POLICY]]
 
-The IDL attribute <dfn attribute for=HTMLFencedFrameElement>allow</dfn> must [=reflect=] the
-respective content attribute of the same name.
+The <dfn element-attr for=fencedframe>sandbox</dfn> attribute, when specified, enables a set of
+extra restrictions on any content hosted by the <{fencedframe}>. Its value must be an [=unordered
+set of unique space-separated tokens=] that are [=ASCII case-insensitive=]. The allowed values are
+the same as the allowed values for the <{iframe}>'s <{iframe/sandbox}> attribute.
+
+When an <{fencedframe}> element's <{fencedframe/sandbox}> attribute is set or changed while it has a
```

Comment:
I think we should use the [attribute changed steps](https://dom.spec.whatwg.org/#concept-element-attributes-change-ext) extension to be more algorithmic. In fact, https://github.com/whatwg/html/pull/10188#discussion_r1719334729 has some good relevant advice on this.

Reply:
Added an "attribute change steps" section for the fencedframe element with handlers for changing/removing the "sandbox" attribute.